docker-for-windows/edge-release-notes.md

```diff
@@ -10,9 +12,39 @@ For information about Stable releases, see the [Stable release
 notes](release-notes). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
-## Edge Releases of 2019
+## Docker Desktop Community 2.1.7.0
+2019-12-11
 
-### Docker Desktop Community 2.1.6.1
+[Download](https://download.docker.com/win/edge/41561/Docker%20Desktop%20Installer.exe)
+
+> **Note:** Docker Desktop Edge 2.1.7.0 is the release candidate for the upcoming major Stable release. Please help us test this version before the wider release and report any issues in [docker/for-win](https://github.com/docker/for-win/issues) GitHub repository.
+
+### Upgrades
+
+- [Docker Compose 1.25.1-rc1](https://github.com/docker/compose/releases/tag/1.25.1-rc1)
+
+### New
+
+- Added the **WSL Integration** option in **Settings** > **Resources** to select WSL 2 distributions. This option will be visible only after enabling WSL 2 on your machine.
+- File sharing: The new gRPC FUSE system is no longer experimental and is enabled by default even when experimental features are turned off.
```

Comment:
Do these acronyms need to be spelled out?

Reply:
No, I don't think so.